core/src/services/s3/backend.rs

```diff
@@ -463,6 +467,26 @@ impl S3Builder {
         self
     }
 
+    /// Set storage_class for this backend.
+    ///
+    /// Available values:
+    /// - `DEEP_ARCHIVE`
+    /// - `GLACIER`
+    /// - `GLACIER_IR`
+    /// - `INTELLIGENT_TIERING`
+    /// - `ONEZONE_IA`
+    /// - `OUTPOSTS`
+    /// - `REDUCED_REDUNDANCY`
+    /// - `STANDARD`
+    /// - `STANDARD_IA`
+    pub fn storage_class(&mut self, v: &str) -> &mut Self {
```

Comment:
How about using `default_storage_class`? I think it's more clear.

Reply:
lgtm